server/webhook_http.go

```diff
@@ -110,5 +110,10 @@ func httpWebhook(p *Plugin, w http.ResponseWriter, r *http.Request) (int, error)
 		return statusCode, err
 	}
 
+	_, statusCode, err = wh.PostNotifications(p)
+	if err != nil {
+		return statusCode, err
+	}
```

Comment:
I noticed the legacy webhook endpoint wasn't posting notifications. I added it in this pr.

Reply:
@dnguy078 FWIW, this is by design - some customers may be using both legacy webhooks and subscriptions, this would lead to potential duplicate notifications.